Rubberduck.Main/Root/RubberduckIoCInstaller.cs

```diff
@@ -152,6 +153,14 @@ public void Install(IWindsorContainer container, IConfigurationStore store)
             ApplyDefaultInterfaceConvention(container, assembliesToRegister);
         }
 
+        private void RegisterUnitTestingComSide(IWindsorContainer container)
```

Comment:
I think we should be consistent in our naming -- If it's in the `API`, it should be `API`, not `ComSide`. 

(OTOH, the `API` is too general a name, so I can see the ambiguity.... Maybe it's the one that needs renaming....)